packages/@ama-sdk/core/src/logger/logger.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Logger interface.
+ */
+export interface Logger {
+  /**
+   * Log an error.
+   * @param message Message to log
+   * @param optionalParams Optional parameters to log
+   */
+  error(message?: any, ...optionalParams: any[]): void;
+
+  /**
+   * Log a warning.
+   * @param message Message to log
+   * @param optionalParams Optional parameters to log
+   */
+  warn(message?: any, ...optionalParams: any[]): void;
+
+  /**
+   * Log a message.
```

Comment:
What's the difference between `info` and `log`?

Reply:
They are aliases, but in some browsers there is a difference in color when displaying the message